### PR TITLE
Use clippy

### PR DESCRIPTION
**Pull request by bigfoodK**
## Where to review
- [Use clippy on cargo-fix-fmt-all.py](https://github.com/NamseEnt/namseent/pull/635/commits/6cab0f876b016fa015346ba5530f369d75faf203).

- [x] Apply clippy to code
- [x] Run clippy on ci
- [x] Discuss about allowing some `clippy::enum_variant_names` and `clippy::too_many_arguments`

### CODE REVIEW

**Comment by bigfoodK** [comment]
I allowed some `clippy::too_many_arguments` and `clippy::enum_variant_names` because it seems better to allow for me.
`clippy::enum_variant_names`
 - [luda-editor/client/src/pages/router.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/router.rs#L44)
 - [luda-editor/client/src/pages/sequence_edit_page/loaded.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/sequence_edit_page/loaded.rs#L51)
 - [luda-editor/client/src/pages/sequence_edit_page/components/cut_list_view/mod.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/sequence_edit_page/components/cut_list_view/mod.rs#L19)
 - [namui-cli/src/cli.rs](https://github.com/NamseEnt/namseent/blob/ca05c928bb181f852e708a9f5bb9ee5607a0ad19/namui-cli/src/cli.rs#L44)

`clippy::too_many_arguments`
- [namui-prebuilt/src/button.rs](https://github.com/NamseEnt/namseent/blob/fa982eeb243eb35cb237abf3c9dcfd7f9057e412/namui-prebuilt/src/button.rs#L26)

@namse what do you think? Is it ok to allow these?

**Comment by namse** [comment]
## clippy::enum_variant_names

[luda-editor/client/src/pages/router.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/router.rs#L44)
[luda-editor/client/src/pages/sequence_edit_page/loaded.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/sequence_edit_page/loaded.rs#L51)

Above cases seems not good to allow that. I think removing `Page` suffix and `Event` makes sense.

[luda-editor/client/src/pages/sequence_edit_page/components/cut_list_view/mod.rs](https://github.com/NamseEnt/namseent/blob/dc84026d8f9ce22262d172868741db2573593bac/luda-editor/client/src/pages/sequence_edit_page/components/cut_list_view/mod.rs#L19)

Prefix is also the same.

[namui-cli/src/cli.rs](https://github.com/NamseEnt/namseent/blob/ca05c928bb181f852e708a9f5bb9ee5607a0ad19/namui-cli/src/cli.rs#L44)

It is different case because we may put other platforms so suffix and prefix doesn't have any pattern for this enum. Other cases had meaningless pattern.


## too_many_arguments

I think many arguments is bad idea for rust because rust don't have named parameter unlike C#. That mean it's very complicated to refactoring arguments order and difficult to know what the parameter is for without rust-analyzer(for example, in github PR review.)

I agree with too_many_arguments warning so maybe we need some good way for this. struct for function or something like that.

Actually we don't need text_button function anymore. We can make enum named `text_button` and put variants like,

```
enum TextButton {
  Normal { ... },
  Fit { ... },
  Align { ... },
}
```

Or just struct for each functions,

```
struct TextButton {
}

struct TextButtonFit {
}
```

So, we don't have to fight with arguments and functions anymore.

**Comment by namse** [comment]
Idea: if you want to allow some clippy cases, create issue and reference it, and put comment with that issue number.

**Review by namse — CHANGES_REQUESTED** [review]

File: .github/actions/init-test-job/action.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ runs:
       shell: bash
       run: cargo install --version 0.11.1 wasm-pack
 
+    - name: clippy install
+      shell: bash
+      run: rustup component add clippy-preview
```
[the remark this answers is not in the document]
Comment (bigfoodK) [reply .github/actions/init-test-job/action.yml:20]: When I tried `cargo install clippy`, [They guided](https://docs.rs/crate/clippy/0.0.302) me to use `rustup component add clippy-preview`.

But I think it was outdated. On [latest guide](https://github.com/rust-lang/rust-clippy) says "just use `rustup component add clippy`".

Let's use `rustup component add clippy`.

**Review by namse — APPROVED** [review]

**Pull request merged**